src/main/resources/templates/admin/employee-edit.html

```diff
@@ -56,8 +79,19 @@
     <br><hr><br>
 </div>
 
-<script src="/webjars/jquery/jquery.min.js"></script>
-<script src="/webjars/bootstrap/js/bootstrap.min.js"></script>
+<script th:src="@{/webjars/jquery/jquery.min.js}"></script>
+<script th:src="@{/webjars/popper.js/umd/popper.min.js}"></script>
+<script th:src="@{/webjars/bootstrap/js/bootstrap.min.js}"></script>
+
 <script src="/webjars/vue/2.6.11/vue.js"></script>
+
+<!-- Trying to add bootstrap-select using webjars: -->
+<!--<script th:src="@{/webjars/bootstrap-select/js/bootstrap-select.min.js}"></script>-->
+<!--<script src="/webjars/bootstrap-select/1.4.2/bootstrap-select.min.js" type="script"></script>-->
+
+<!-- Using CDN is ok: -->
```

Comment:
```suggestion
<!-- Using CDN is ok: -->
```
Se temos certeza que o cliente está conectado a internet. Ou seja, é ok para a demo, mas e pra uma instalação on-premisses?

Reply:
Com CDN deu certo então deixei, mas concordo que o ideal é ter local mesmo. 
Mas achei meio tentativa-e-erro descobrir qual é o path até o asset, porque, o mesmo padrão não funcionou para o bootstrap-select

Reply:
Eu uso uma dependência chamada webjars-locator e com ela você pode omitir o número da versão do Path dos webjars.

E sobre saber o caminho, minha IDE completa e depois eu vou lá e removo a versão. Imagino que fique mais fácil atualizar bibliotecas de front assim pois só no pom a versão aparece.

Reply:
Vamos fazer o seguinte: vamos usar cdn por padrão... pra esse repo que é uma demonstração funcional acho que vai ser sempre verdade o deployment na nuvem.

Reply:
> E sobre saber o caminho, minha IDE completa e depois eu vou lá e removo a versão. Imagino que fique mais fácil atualizar bibliotecas de front assim pois só no pom a versão aparece.
Você precisou instalar algum plugin pra isso? Porque no intellij *commmunity* out-the-box ele não autocompleta nada, na verdade, o código thymeleaf fica até meio error-prone por causa disso... tipo, se add um parametro novo no template e não passar ele, o template quebra (obvio) mas não tem nenhum highlight de error, por ex. :crying_cat_face: